pages/index.tsx

```diff
@@ -1,39 +1,33 @@
-import type { NextPage } from 'next'
-import Head from 'next/head'
-import { ConnectWallet } from '../components/metamask'
-import { Container, Content, Main, Tiltle } from '../components/common'
-
+import type { NextPage } from "next";
+import Head from "next/head";
+import { ConnectWallet } from "../components/metamask";
+import { Container, Content, Main, Title } from "../components/common";
 
 const Home: NextPage = () => {
   return (
-    <body style={{background: 'rgb(15 19 22)'}}>
+    <div style={{ background: "rgb(15 19 22)" }}>
       <Container>
         <Head>
-          <title>Henkaku Shiniri</title>
+          <title>HENKAKU Shiniri</title>
```

Comment:
👀

Reply:
I'm propagating the fact that isbtty was particular about this. 😃